tests/spec/array/spec-repeat.js

```diff
@@ -0,0 +1,17 @@
+define(['mout/array/repeat'], function (repeat) {
+
+  describe('array/repeat()', function(){
+
+    it('should create an array of size 0.', function(){
+      expect(repeat(0, 'a')).toEqual([]);
+    });
+
+    it('should create an array of size 1 and fill with a value "a".', function(){
+      expect(repeat(1, 'a')).toEqual(['a']);
+    });
+
+    it('should create an array of size N and fill with a value "a".', function(){
+      expect(repeat(2, 'a')).toEqual(['a', 'a']);
+    });
```

Comment:
Would you mind also adding some tests with longer strings as well as numbers?

Reply:
Added a test for the longer string and using a number.